src/barril/units/posc.py

```diff
@@ -13317,6 +13318,116 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
         f_unit_to_base,
         default_category=None,
     )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 1.0, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 1.0, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "degC/Pa",
+        "degC/Pa",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 0.00001, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 0.00001, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "degC/bar",
+        "degC/bar",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 1.0e6, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 1.0e6, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "degC/MPa",
+        "degC/MPa",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 0.00001, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 0.00001, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "K/bar",
+        "K/bar",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 1.0, 1.0, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 1.0, 1.0, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "K/MPa",
+        "K/MPa",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 5.0, 9.0, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 5.0, 9.0, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "degF/Pa",
+        "degF/Pa",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 5.0, 9.0e-5, 0.0)
+    f_base_to_unit = MakeBaseToCustomary(0.0, 5.0, 9.0e-5, 0.0)
+    db.AddUnit(
+        "temperature per pressure",
+        "degF/bar",
+        "degF/bar",
+        f_base_to_unit,
+        f_unit_to_base,
+        default_category=None,
+    )
+    f_unit_to_base = MakeCustomaryToBase(0.0, 5.0, 9.0e6, 0.0)
```

Comment:
Should it be `9.0e-6` here?

Reply:
I'm really confused with these factors. Since `1 MPa = 1e6 Pa`, the value should be `1e6`, or not? Now I have I feeling that and inverted everything in my mind.

Reply:
I mean, when you want to convert `MPa -> Pa`, you should multiplicate it by `1e6`. I'm sorry, but could you please clarify it to me? The values `a, b, c, d` in the documentation are a little bit confusing for me :/

Reply:
```
def MakeCustomaryToBase(a, b, c, d):
    """
    Formula to convert some value from Customary Unit to Base Unit
    (A + BX) / (C + DX)
    X can be any single numeric value or a numpy array
    Integer values are correctly handled, once a, b, c and d parameters
    are always floats
```

you can multiply by 1e6 or divide by 1e-6. Since `C` is being changed (in the denominator), it looks like it should be `9e-6` here.